documentation/user_docs/docs/manual/Cutting_data_of_interest_from_SQW_files_and_objects.rst

```diff
@@ -297,290 +344,610 @@ the position of the plain you want to cut. These fields are ``u`` and ``v``:
 
-   Spherical coordinate system used by ``sphere_proj`` in case of orthogonal and non-orthogonal reciprocal lattice. In latter case vectors :math:`e_x,e_z` define the plane where orthogonal to :math:`e_z` x-axis (:math:`r_x` on the picture) of spherical coordinate system lies.
+   Spherical coordinate system used by ``sphere_proj``.
 
 
 .. note::
 
-   A spherical projection does not have any scaling aspect to the
-   |Q| in the same way a ``line_proj`` can define non-unitary
-   vectors as the axes.
+   A spherical projection currently does not have the ability to be
+   rescaled in |Q| relative to the magnitude of :math:`e_x` or
+   :math:`e_z`.
 
-When it comes to cutting and plotting, we can use a ``sphere_proj`` in exactly
-the same way as we would a ``line_proj`` with one key difference. The binning
-arguments of ``cut`` no longer refer to :math:`h,k,l,E`, but to |Q|,
-:math:`\theta`, :math:`\phi`, :math:`E` coordinates.
+When it comes to cutting and plotting, we can use a ``sphere_proj`` in
+exactly the same way as we would a ``line_proj``, but with one key
+difference. The binning arguments of ``cut`` no longer refer to
+:math:`h,k,l,E`, but to |Q|, :math:`\theta`, :math:`\phi`, :math:`E`.
 
 .. code-block:: matlab
 
    sp_cut = cut(w, sp_proj, Q, theta, phi, e, ...);
 
-The structure of the arguments to cut is still the same (see `Binning arguments`_)
+.. warning::
+
+   The form of the arguments to ``cut`` is still the same (see: `Binning
+   arguments`_), however:
 
-By default a ``sphere_proj`` will define its principal axes :math:`e_z` and :math:`e_x` or rather 
-:math:`r_x` for angular integration (:math:`\theta`, :math:`\phi`) along 
-defined :math:`hkl` directions :math:`[1,0,0] (e_z)` and :math:`[0,1,0] (e_x)`. Using ``sphere_proj`` 
-properties :math:`e_z` and :math:`e_x` spherical coordinate system may be reoriented to any
-reciprocal lattice direction. Naturally, :math:`e_z` and :math:`e_x` vectors can not be parallel. 
-   
-   
-``sphere_proj`` 2D and 1D cuts samples:
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-Spherical projection allows you to obtain powder average. Note that binning ranges are specified in 
-target coordinate system. Energy transfer by default is expressed in inverse Angstroms and angles 
-are in degrees.
+   - |Q| runs from :math:`[0, \infty)` (attempting to use a |Q| with a minimum
+     bound less than :math:`0` will result in unexpected behaviour)
+   - :math:`\theta` runs between :math:`[0, 180)`
+   - :math:`\phi` runs between :math:`[-180, 180)`
+   - Attempting to use default binning (``[]``, ``[n]``) will produce
+     erroneous results if source object is in a linear projection
+     (``line_proj``).
+
+
+``sphere_proj`` 2D and 1D cuts examples:
+________________________________________
+
+By integrating over the angular terms of a spherical projection you
+can obtain the powder average of a sample (excluding certain powder
+effects if done on a single crystal).
```

Comment:
Are you sure? I am not aware of any powder effects excluded. I thougt you are building excact powder average. 
The effects should be either clarified or sentence removed.

Reply:
This comment came I believe out of a discussion with Russell will double check at lunch

Reply:
I think a clearer wording would be:

```
Integrating over the angular terms of a spherical projection of a
**single crystal** dataset will give an **approximation** of a powder
average of the sample. This is because (except for low scattering
angles) the detectors do not cover the full 4*pi solid angular range.
Thus regions without detector coverage will not be sampled by
the angular spherical integration. In contrast for a true powder
sample, there will be crystal grains with the correct orientation to
be detected even by the limited detector coverage.
Integrating over the angular terms for a *powder* sample is
a valid powder averaging.

At low scattering angles (below approximately 30 degrees on LET),
the detectors *do* cover the full angular range so the angular
integration of a single crystal dataset will give a valid powder
average.

These effects are important to bear in mind when modelling the
scattering - e.g. for a single crystal dataset it is best to model it
as a single crystal and then let Horace perform the angular
integration, rather than treating it as powder data.
```

Sorry it's a bit long-winded... perhaps you can make it terser.